nuxeo-features/nuxeo-admin-center/nuxeo-connect-update/src/main/java/org/nuxeo/connect/update/task/live/commands/Undeploy.java

```diff
@@ -78,6 +89,32 @@ protected Command doRun(Task task, Map<String, String> prefs) throws PackageExce
             log.warn("Can't undeploy file " + file + ". File is missing.");
             return null;
         }
+        boolean useFormerReload = Framework.isBooleanPropertyTrue(ReloadService.USE_FORMER_HOT_RELOAD);
+        if (useFormerReload) {
+            doFormerRun(task);
+        }
```

Comment:
Shouldn't there be a `return` here? If not, then please add a comment explaining that we don't do it on purpose.